logins-api/android/build.gradle

```diff
@@ -30,6 +30,10 @@ buildscript {
 
         classpath 'gradle.plugin.org.mozilla.rust-android-gradle:plugin:0.2.1'
 
+        // Yes, this is unusual.  We want to access some host-specific
```

Comment:
Great work following the Mentat pattern.  Glad you got this all up and running!

Reply:
Lots of comments copied over from mentat, oh well...

Reply:
No, this is good -- those comments are high value and make it easy to search for the corresponding pieces across projects.